addons/sys_gui/XEH_missionDisplayLoad.sqf

```diff
@@ -3,4 +3,4 @@
 params ["_display"];
 
 // Init the volume control.
-GVAR(MWheel) = _display displayAddEventHandler ["MouseZChanged", QUOTE(_this call FUNC(onVolumeControlAdjust) )];
\ No newline at end of file
+GVAR(MWheel) = _display displayAddEventHandler ["MouseZChanged", {call FUNC(onVolumeControlAdjust)}];
```

Comment:
`_this` is missing, this is common theme throughout this PR does this actually still pass `_this` into the function?